cirq/google/sim/xmon_stepper.py

```diff
@@ -346,7 +346,7 @@ def simulate_measurement(self, index: int) -> bool:
             index: Which qubit is measured.
 
         Returns:
-            True iff the measurement result corresponds to the |1> state.
```

Comment:
Wait, why was this a warning?

Reply:
Ah sorry, this was not a warning, just a typo that I noticed while going through the code.

Reply:
This isn't a typo, `iff` is a mathematical shorthand for "if and only if". By removing the extra f, the docstring no longer precludes the possibility that you could also return `True` if the result corresponds to the |0> state :)

Reply:
Oh, I see... Today I learned, thanks! Reverted back.